internal/dag/cache.go

```diff
@@ -439,6 +439,16 @@ func (kc *KubernetesCache) serviceTriggersRebuild(service *v1.Service) bool {
 		}
 	}
 
+	for _, route := range kc.grpcroutes {
+		for _, rule := range route.Spec.Rules {
+			for _, backend := range rule.BackendRefs {
+				if isRefToService(backend.BackendObjectReference, service, route.Namespace) {
+					return true
+				}
+			}
+		}
+	}
+
```

Comment:
Please add some unit test cases to `TestServiceTriggersRebuild` for GRPCRoute following the existing test pattern.

Reply:
Thanks for quick review.
I will do it later.